lib/manager/pre-commit/index.ts

```diff
@@ -2,5 +2,9 @@ export { extractPackageFile } from './extract';
 
 export const defaultConfig = {
   commitMessageTopic: 'pre-commit hook {{depName}}',
+  enabled: false,
   fileMatch: ['(^|/)\\.pre-commit-config\\.yaml$'],
+  prBodyNotes: [
+    'Note: The `pre-commit` manager in Renovate is not supported by the `pre-commit` maintainers or community. Please do not report any problems there, instead [create a Discussion in the Renovate repository](https://github.com/renovatebot/renovate/discussions/new) if you have any questions.',
+  ],
```

Comment:
I was wondering, do we need a new/adjusted unit test for this piece of code?

Reply:
No, it's just config